src/main/content/engine/engine-content.ts

```diff
@@ -89,19 +89,19 @@ export class EngineContentAPI extends AbstractContentAPI<string, EngineVersion>
             if (this.isVersionInstalled(engineVersion)) {
                 return;
             }
-            const { data } = await this.ocotokit.rest.repos.getReleaseByTag({
-                owner: contentSources.engineGitHub.owner,
-                repo: contentSources.engineGitHub.repo,
-                tag: engineVersion,
-            });
-            if (!data) {
+
+            const archStr = process.platform === "win32" ? "windows64" : "linux64";
+            const engineResponse = await axios.get(`https://files-cdn.beyondallreason.dev/find?category=engine_${archStr}&springname=${engineVersion}`);
+
+            if (engineResponse.status !== 200) {
```

Comment:
As far as I understand from axios docs, by default it throws error when error code is not 2XX, this won't trigger.